cmd/aperturectl/cmd/utils/controller.go

```diff
@@ -174,10 +174,6 @@ func (c *ControllerConn) PreRunE(_ *cobra.Command, _ []string) error {
 			return fmt.Errorf("invalid config file '%s'. Missing key 'controller.url'", c.config)
 		}
 
-		if config.Controller.APIKey == "" {
```

Comment:
Are we removing this due to some reason? I think it is a mandatory field for cloud controller communication?

Reply:
Yes, but realized that config files can be utilized also for self-hosted controller, so it's ok to not have api key. Feel free to revert it if you disagree